packages/components/src/index.js

```diff
@@ -1,7 +1,7 @@
 // Components
 export * from './primitives';
```

Comment:
This thing overrides `Svg` from `primitives` now :)

We need to find out what would be the best strategy for maintaining `Svg` to make it work for both web and mobile (RN).

Reply:
yes, maybe let's keep it as `AccessibleSVG` for now.